components/datatypes/contentaudit.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/contentaudit",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Audit",
+  "type": "object",
+  "meta:abstract": true,
+  "description": "Audit information about digital content such as a digital experience or asset.",
+  "definitions": {
+    "content-audit": {
+      "properties": {
+        "xdm:contentAudit": {
+          "title": "Content Audit",
+          "description": "Audit information about digital content such as a digital experience or asset.",
+          "type": "object",
+          "properties": {
+            "xdm:createdBy": {
+              "title": "Created By",
+              "description": "Name of the user who created this record",
+              "type": "string"
+            },
+            "xdm:createdDate": {
+              "title": "Created Date",
+              "description": "Date when this record was created.",
+              "type": "string",
+              "format": "date-time"
+            },
+            "xdm:lastUpdatedDate": {
+              "title": "Last Updated Date",
+              "description": "Date when this record was last modified.",
+              "type": "string",
+              "format": "date-time"
+            },
+            "xdm:lastUpdatedBy": {
+              "title": "Last Updated By",
+              "description": "User who last modified this record.",
+              "type": "string"
```

Comment:
should this description be "user" or "service"

Reply:
https://github.com/adobe/xdm/pull/1942/files